src/clj/rems/api/resources.clj

```diff
@@ -38,7 +38,7 @@
 
     (GET "/" []
       :summary "Get resources"
-      :roles #{:owner :handler}
+      :roles #{:owner :organization-owner :handler}
```

Comment:
if we add `:organization-owner` access before we implement the restrictions, the organization-owner feature is broken (= reveals more information than it should) in the interim. perhaps the roles should only get added to the endpoints at the same pace as the organization-owner access restrictions are implemented?

Reply:
Done.